documentation/get-started/certification.en.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Certification
+tx_slug: documentation_get-started_certifiation
+---
+
+# Certification
+
+The QField Certificate Program is designed to foster community involvement in the QField project and provide high-quality education for the QField application. Therefore, the application process requires applicants to detail their contributions to the QField project and make their training materials available for review. Activities that contribute to the QField and QFieldCloud project include:
+
+Contributions to the QField documentation and training materials
+
+Translation of QField materials
+
+Filing bug reports
+
+Financially supporting new feature development and sponsorship of QField
+
+Involvement in your local QField User Group
+
+Publication of open QField courseware
+
+Contributing to QField/QFieldCloud code
+
+ To reach out to the certification team, please visit the next page at https://qfield.org/certified-trainer.html. 
+
+## Certification Process
+
+For each of your courses, you are required to provide training materials. This includes exercises, lectures, and data. This material will be reviewed to ensure that it is current, accurate, and high-quality. If the material is not of adequate quality, it may be refused.
+
+OPENGIS.ch will handle the review.
+
+If accepted as a QField Certified Organization, your contributions will be published for transparency.
+
+If approved, you must pay a €20 certification fee to the QField project for each certificate you issue. This establishes a mechanism for financial support for the project. Certificate payments are made using online credit card transactions or, on request, by regular invoice and SEPA wire transfer. Payments can be handled in batches for many certificates at once.

```

Comment:
If your application is successful and you become a QField Certified Organization, your contributions will be publicly disclosed to ensure transparency.

Upon approval, a certification fee of €20 per certificate issued will be required to support the QField project financially. 
This fee can be conveniently settled through online credit card transactions or, upon request, via regular invoice and SEPA wire transfer.
Additionally, payments for multiple certificates can be processed together for your convenience.